osquery/CMakeLists.txt

```diff
@@ -36,11 +36,23 @@ set(OSQUERY_LIBS
   ${OPENSSL_SSL_LIBRARY}
   ${READLINE_LIBRARIES}
 
-  pthread
-  bz2
-  z
 )
 
+if(WIN32)
+  set(OSQUERY_LIBS 
```

Comment:
So, I dug in a bit more as to why we have `boost_thread_library`. It appears that one of our dependencies, `thriftmd.lib` depends upon it (`boost::this_thread::interruptible_wait`).